synapse/http/client.py

```diff
@@ -730,7 +730,7 @@ def _timeout_to_request_timed_out_error(f: Failure):
     return f
 
 
-class _ReadBodyToFileProtocol(protocol.Protocol):
+class readBodyToFileProtocol(protocol.Protocol):
```

Comment:
class names start with capital letters.

Reply:
Ugh, I didn't actually mean to rename this. There's no reason for it to be public. Shoddy work, sorry about that.